__tests__/release.test.ts

```diff
@@ -0,0 +1,47 @@
+import {listReleases} from '../src/release'
+import {DownloadRelease, Version} from '../src/types'
+import {expect, test, jest} from '@jest/globals'
+
+test('get download releases', async () => {
+  const mock = jest.fn()
+  mock.mockReturnValueOnce(
+    new Promise(resolve => {
+      resolve([
+        {
+          tag_name: 'v0.0.1'
+        },
+        {
+          tag_name: 'v0.0.2'
+        },
+        {
+          tag_name: 'v1.0.0'
+        },
+        {
+          tag_name: 'v1.0.1'
+        },
+        {
+          id: 12345,
+          tag_name: 'download-latest-v0',
+          body: '# Download v0\n\nThis release tracks the latest v0 available, currently v0.0.1.'
+        },
+        {
+          id: 67890,
+          tag_name: 'download-latest-v1',
+          body: '# Download v0\n\nThis release tracks the latest v0 available, currently v1.0.1.'
```

Comment:
Should the body contain `This release tracks the latest v0 ...`? I guess it doesn't matter for the test.

Reply:
The body should only contains a reference to the related tag in the `vX.Y.Z` format.
Otherwise, the body could be anything.

Reply:
Ok, I was just commenting on the confusing fact that the `download-latest-v1` tag had `latest v0` in the body.